docs/pages/blog/mui-core-v5.md

```diff
@@ -531,8 +531,23 @@ It's new, link core and x roadmaps.
 
 ### Core
 
-- Unstyled
-- Joy
+#### Unstyled components and hooks
+
```

Comment:
A small description of what are the unstyled components or at least a link to a place introducing them could be nice